Projeto-Stream/src/views/Home/homeDetail.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+    <div :style="{ backgroundImage: `linear-gradient(to right, black, rgba(0, 0, 0, 0.4)), url(${backgroundUrl})` }"
+        class="bg-fixed bg-center bg-cover h-screen py-24 px-10 grid items-center ">
+        <div>
+            <div class="p-5 w-12/12 md:w-4/12">
+                <h1 class="text-3xl font-base font-bold whitespace-normal text-white">{{ nameOrTitle }}
+                </h1>
+                <div class="flex gap-4 items-center my-2 text-white text-sm">
+                    <p class="font-base font-medium ">{{ firstAirDate }}</p>
+                    <p class="text-green-700 font-base font-bold">{{ averageVote }}</p>
+                    <p class=" font-base font-medium">|</p>
+                    <p class="  font-base font-medium">{{ movieOrTvShow?.runtime }} min</p>
+                </div>
+                <div class="text-white mb-2 flex flex-col gap-2">
+                    <h3 class="text-lg font-bold">{{ movieOrTvShow?.tagline }}</h3>
+                    <p class="text-sm font-base font-medium line-clamp-6 text-justify">{{ movieOrTvShow?.overview }}</p>
+                </div>
+                <div class="text-white py-5" >
+                    <prime-button @click="setItemLocalStorage" class="border-2 border-white rounded-lg 
+                    px-4 py-2 shadow-md cursor-pointer transform transition duration-200 hover:shadow-2xl 
+                    hover:scale-95 flex items-center font-bold" label="Favoritar" icon="pi pi-heart-fill " />
+                </div>
+            </div>
+
+        </div>
+    </div>
+</template>
+<script lang="ts">
+import { QueryParamsModel } from '@/model/queryParams.model';
+import { HomeService } from './home.service';
+import type { MovieAndTvShow } from '@/model/movieAndTvShow.model';
+import LocalStorage from '@/utils/localStorage.utils';
+
+export default {
+    data() {
+        return {
+            backgroundBaseUrl: "https://image.tmdb.org/t/p/original/",
+            movieOrTvShow: {} as MovieAndTvShow,
+            favorites: [] as MovieAndTvShow[]
+        }
+    },
+    methods: {
+        getMovieAndTvShowById(queryParams: QueryParamsModel) {
```

Comment:
add the return type to each method. This applies to all methods and variables computed in the project.